lib/saluki-components/src/sources/checks/scheduler.rs

```diff
@@ -72,7 +89,7 @@ impl Scheduler {
             self.start_interval_ticker(interval_secs);
         }
 
-        debug!("Scheduled check '{}' with interval {}s", check_id, interval_secs);
+        info!("Scheduled check '{}' with interval {}s", check_id, interval_secs);
```

Comment:
```suggestion
        info!(check_id, check_interval_secs = interval_secs, "Scheduled periodic check.", check_id);
```

We generally try to prefer using structured key/value pairs, because it lets us get structured logs for free.